lessons/Early-Math/translating/langs/en-us/pages/matching-math-to-words.adoc

```diff
@@ -0,0 +1,46 @@
+= Matching Math to Words
+
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Draw a line from the words on the left to the Circle on the right. *Some Circles have more than one correct translation.*
+
+[.FillVerticalSpace, cols="^.^10a,^.^3a,5a,^.^1a,^.^10a", options="header", stripes="none", grid="none", frame="none"]
+|===
+| Math
+|||
+| Words
+
+| @math{(\frac {1}{3} \times 10) + 2}
+|*@n*@showsoln{"-C" }||*A*
+| double one-third of 10
+
+| @math{2 \times (\frac {1}{3} \times 10)}
+|*@n*@showsoln{"-A" }||*B*
+| increase half of 10 by one-third
+
+| @math{(10 \div 2) + \frac {1}{3}}
+|*@n*@showsoln{"-B" }||*C*
+| increase one-third of 10 by 2
+
+| @math{(10 + \frac {1}{3}) \times 2}
+|*@n*@showsoln{"-F" }||*D*
+| the sum of 10 and 2, decreased by one-third
+
+| @math{2 + (10 - \frac {1}{3})}
+|*@n*@showsoln{"-E" }||*E*
+| 2 increased by the difference of 10 and one-third
+
+| @math{(2 + 10) - \frac {1}{3}}
+|*@n*@showsoln{"-D" }||*F*
+| the sum of 10 and one-third, doubled
+
+
+
+|===
+
```

Comment:
Most of the mathematical expressions included on this page use unnecessary parentheses that make it so that students do not have to think about whether addition or multiplication comes first. Is that intentional? It's something I've worked hard to avoid providing in materials I've developed.

Reply:
I will fix! Thanks.